ezcontent_admin.info.yml

```diff
@@ -137,10 +137,9 @@ libraries-override:
       theme:
         css/paragraphs_ee.off_canvas.css: false
 
-  gin/gin:
+  gin/dist:
```

Comment:
Please add a comment stating why we are doing this.

Reply:
iwe used the gin.css ezcontent_admin theme for style the our theme, but every-time when gin theme will be updated the version, we have to replace the gin.css by update one in our admin theme. but we have remove it

Reply:
Please add this reason as a comment in the code, for future reference.

Reply:
Done